classes/Page/Page.mjs

```diff
@@ -90,12 +91,15 @@ export default class Page {
     }
 
     /**
-     * Check the Project for any RERUM documents and either upgrade a local version or overwrite the RERUM version.
-     * @returns {Promise} Resolves to the updated Layer object as stored in Project.
-     */
-    async update() {
-        if (this.#tinyAction === 'update' || this.items.length) {
-            await this.#setRerumId().#savePageToRerum()
+      * Check the Project for any RERUM documents and either upgrade a local version or overwrite the RERUM version.
+      * @returns {Promise} Resolves to the updated Layer object as stored in Project.
+      */
+    async update(userId) {
+        const hasItems = Array.isArray(this.data?.items) && this.data.items.length > 0
```

Comment:
What are these changes for? I'm not sure it wasn't working before. Page.update shouldn't save itself to the Project, just RERUM.